packages/hoppscotch-app/components/http/ImportCurl.vue

```diff
@@ -53,7 +55,17 @@ useCodemirror(curlEditor, curl, {
   environmentHighlights: false,
 })
 
-defineProps<{ show: boolean }>()
+watch(
+  () => props.show,
+  () => {
+    if (props.show) {
+      setTimeout(() => {
+        curl.value = props.text || ""
+      }, 100)
+    }
+  },
+  { immediate: false }
+)
```

Comment:
Yeah, that's good. But, the problem is props.text containing previous value instead of updated value. that's why I added `setTimout` so that props get updated meanwhile. I think there must be a better way to do this. Could you please suggest me how can I accomplish this?